tests/kernel/pthread/src/pthread.c

```diff
@@ -181,7 +172,34 @@ int barrier_test_done(void)
 
 void main(void)
 {
-	int status = TC_FAIL;
+	int i, ret, status = TC_FAIL;
+	pthread_attr_t attr[N_THR];
+	struct sched_param schedparam;
+	pthread_t newthread[N_THR];
+	int schedpolicy = SCHED_FIFO;
+
+	schedparam.priority = CONFIG_NUM_COOP_PRIORITIES - 1;
+	if ((schedparam.priority < sched_get_priority_min(schedpolicy)) ||
+	    (schedparam.priority > sched_get_priority_max(schedpolicy))) {
+		TC_START("Scheduling priority outside valid priority range\n");
+		return;
+	}
+
+	for (i = 0; i < N_THR; i++) {
+		/* Thread Attribute initialization */
```

Comment:
Please avoid comments that are not adding any value. If a line says `foo_init(&foo)`, don't say `/* Initializes foo */`. Comments should document the intent of the code or things that are not obvious just from reading the code itself.